code-challenges.js

```diff
@@ -1,64 +1,117 @@
 // ASSESSMENT 2: Coding Practical Questions
 
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
-// Use the test variables provided.
-
-var num1 = 15
-// Expected output: "15 is divisible by three"
-
-var num2 = 0
-// Expected output: "0 is divisible by three"
-
-var num3 = -7
-// Expected output: "-7 is not divisble by three"
-
-
-
-
+// // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+// // Use the test variables provided.
+
+// var num1 = 15;
+// // Expected output: "15 is divisible by three"
+// var num2 = 0;
+// // Expected output: "0 is divisible by three"
+// var num3 = -7;
+// // Expected output: "-7 is not divisble by three"
+
+
+// // i got the concept. but the problem is that it only checks for the first argument and then goes answers num1 and never checks for num2. it is 3:28 pm on sunday. i will go on and then come back later to that problem :) time spent: 4,5 hrs
+
+
+// function divisibleNum(num) {
+//     if (num % 3 === 0){
+//         return `${num1} is divisible by three`;
+//     } if  (num === 0 ){
+//         return `${num2} is divisible by three`;
+//     } else if (num % 3 < 0 ){
+//         return `${num3} is not divisible by three`;
+//     }   else {
+//         return "error";
+//     }
```

Comment:
super close! think about this: you have a condition where is num divided by 3 has a remainder of 0, we know its divisible by 3 which is great.  so we can assume that if num divided by 3 does NOT have a remainder of 0, it must not be divisible by 3. how do we write that in code form? once you have that written in code, it could be your `else if`!